icu4c/source/test/intltest/numbertest_api.cpp

```diff
@@ -1391,6 +1391,30 @@ void NumberFormatterApiTest::roundingOther() {
             u"1",
             u"1",
             u"0");
+
+    assertFormatSingle(
+            u"ICU-20974 Double.MIN_NORMAL",
+            u"scientific",
+            u"E0",
+            NumberFormatter::with().notation(Notation::scientific()),
+            Locale::getEnglish(),
+            DBL_MIN,
+            u"2.225074E-308");
+
+#ifndef DBL_TRUE_MIN
+#define DBL_TRUE_MIN 4.9E-324
+#endif
+
+    // Note: this behavior might change after upstream #126 gets resolved.
+    // https://github.com/google/double-conversion/issues/126
+    assertFormatSingle(
+            u"ICU-20974 Double.MIN_VALUE",
+            u"scientific",
+            u"E0",
+            NumberFormatter::with().notation(Notation::scientific()),
+            Locale::getEnglish(),
+            DBL_TRUE_MIN,
+            u"5E-324");
```

Comment:
In C++, this is "5E-324", in Java it's "4.9E-324".  Is that a limitation of C++ arithmetic somehow?  Is that okay and/or should that be documented in the Jira ticket / elsewhere?

Reply:
The comment immediately above this test, added in this PR, does exactly that, citing an upstream issue.

Reply:
Totally missed that.  Thanks for the info.  Looks like they closed the issue as WAI, so I guess the comment could be touched up accordingly, but otherwise LGTM.